meta/store.go

```diff
@@ -241,6 +241,11 @@ func (s *Store) createLocalNode() error {
 	return nil
 }
 
+// IsLeader returns true if this node is currently the leader.
```

Comment:
@benbjohnson has a different, simpler, way to do this in his PR. As do I. We all needed it.